lib/msf/core/payload/php.rb

```diff
@@ -16,15 +16,16 @@ module Msf::Payload::Php
   #
   # @return [String] A chunk of PHP code
   #
-  def php_preamble(options = {})
-    dis = options[:disabled_varname] || '$' + Rex::Text.rand_text_alpha(rand(4) + 4)
-    dis = '$' + dis if (dis[0,1] != '$')
+  def self.preamble(options = {})
+    vars = options.fetch(:vars_generator) { Rex::RandomIdentifier::Generator.new(language: :php) }
 
-    @dis = dis
+    dis = options[:disabled_varname] || vars[:disabled_varname]
+    dis    = '$' + dis unless dis.start_with?('$')
```

Comment:
Is this expected to be misaligned like this?
I would think rubocop would have caught this, but maybe it gets aligned with the `=` further down in the file?